docs/upgrading.rst

```diff
@@ -101,7 +101,8 @@ Import BaseClient from globus_sdk
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
We should update the title for this section. Maybe
> Updates to BaseClient Usage

?

Reply:
Good suggestion, I'll use that.